build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1065,14 +1065,15 @@ message OutputFile {
 // [Directory][build.bazel.remote.execution.v2.Directory] protos in a
 // single directory Merkle tree, compressed into one message.
 message Tree {
-  // The root directory in the tree.
-  Directory root = 1;
+  reserved 1, 2; // Used for removed fields in an earlier version of the API.
```

Comment:
I assume we need to keep this field in for backward compatibility with existing implementations.

Reply:
I'm not 100% sure on how these sorts of migrations are supposed to happen, but at some point you have to drop the previous definitions - if we come to a semver/requested version interaction conclusion here, then I suppose so, but otherwise   I believe the fields should just be dropped.